llvm/test/tools/llvm-exegesis/RISCV/latency-by-load.s

```diff
@@ -0,0 +1,60 @@
+# RUN: llvm-exegesis -mode=latency -mtriple=riscv64-unknown-linux-gnu --benchmark-phase=assemble-measured-code -opcode-name=LD | FileCheck --check-prefix=TEST1 %s
```

Comment:
nit: usually all RUN lines would be put at the very top of the file. Also, could you use the opcode you're testing as FileCheck prefix rather than `TESTX`?

Reply:
Changed prefix in FileCheck to opcode name, thanks. I prefer to leave RUN near the CHECK lines. It is more logical and easy to read for me